lib/trip_dispatcher.rb

```diff
@@ -90,6 +91,56 @@ def load_trips
       trips
     end
 
+    def find_last_trips
+      last_trips = []
+      drivers.each do |driver|
+        unless driver.trips.empty? || driver.trips.first.end_time == nil
```

Comment:
I love how you've broken this logic out into a separate helper method. This is an excellent example of functional decomposition, and really helps make the code below easier to read.